features/lorawan/system/LoRaWANTimer.h

```diff
@@ -0,0 +1,115 @@
+/**
+ / _____)             _              | |
+( (____  _____ ____ _| |_ _____  ____| |__
+ \____ \| ___ |    (_   _) ___ |/ ___)  _ \
+ _____) ) ____| | | || |_| ____( (___| | | |
+(______/|_____)_|_|_| \__)_____)\____)_| |_|
+    (C)2013 Semtech
+
+Description: Timer objects and scheduling management
+
+License: Revised BSD License, see LICENSE.TXT file include in the project
+
+Maintainer: Miguel Luis and Gregory Cristian
+
+
+Copyright (c) 2017, Arm Limited and affiliates.
+
+SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef MBED_LORAWAN_SYS_TIMER_H__
+#define MBED_LORAWAN_SYS_TIMER_H__
+#include "drivers/Timer.h"
+#include "drivers/Ticker.h"
+
+using namespace mbed;
```

Comment:
please don't in the header file, shall be removed

Reply:
If not here then where ? Timer and Ticker are defined under namespace mbed.

Reply:
use `namespace mbed { and define your things here }`, not using namespace in the header file, leads to namespace pollution